sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -149,38 +364,38 @@ public EncryptedBlobClientBuilder endpoint(String endpoint) {
         return this;
     }
 
-    @Override
-    protected void addOptionalEncryptionPolicy(List<HttpPipelinePolicy> policies) {
-        BlobDecryptionPolicy decryptionPolicy = new BlobDecryptionPolicy(keyWrapper, keyResolver);
-        policies.add(decryptionPolicy);
-    }
-
     /**
-     * Sets the name of the container this client is connecting to.
-     * @param containerName the name of the container
-     * @return the updated EncryptedBlobClientBuilder  object
+     * Sets the name of the container that contains the blob.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.specialized.BlobClientBase.Builder.containerName#String}
```

Comment:
The codesnippet tags need to be updated to reflect new classnames in all builders. 

In this instance, do we really need code sample on using a setter method?

Reply:
Removed